spec/features/flights_index_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
```

Comment:
Remember for this file structure, it needs to match the file structure of your views folder. So this path should be `spec/features/flights/index_spec.rb`